pyart/retrieve/tests/compare_kdp_proc.py

```diff
@@ -0,0 +1,192 @@
+""" Unit tests for pyart.retrieve.kdp_proc module. """
```

Comment:
This file is not a unit test so should not be labeled as such.  How about:

A script for comparing the e Vulpiani, Maesaka and Schneedbeli methods for KDP estimation with sample data.